[32mINFO    [0m request_logger:request_helper.py:28 Request: GET https://jsonplaceholder.typicode.com/invalidendpoint
[32mINFO    [0m request_logger:request_helper.py:29 Request Headers: {'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Type': 'application/json'}
[32mINFO    [0m request_logger:request_helper.py:30 Request Body: None
[32mINFO    [0m request_logger:request_helper.py:31 Response Status: 404
[32mINFO    [0m request_logger:request_helper.py:32 RESPONSE: 404 for https://jsonplaceholder.typicode.com/invalidendpoint
[35mDEBUG   [0m request_logger:request_helper.py:33 Response Body: {}
[32mINFO    [0m tests.test_error_cases:test_error_cases.py:44 *******
[32mINFO    [0m tests.test_error_cases:test_error_cases.py:45 {}